### ISSUE TITLE
Video Quality

### ISSUE THREAD

**Issue by pascalWeisbrod** [joschahenningsen/TUM-Live#1250]
**Is your feature request related to a problem? Please describe.**

In the libraries of TUM the Wifi is not always the fastest. Especially, when there's a lot of other students logged into the network. The streams are very hard to follow when it keeps buffering or does not download the video fast enough. 

**Describe the solution you'd like**

A way of changing video quality, too make the stream download faster and make it smoother to follow. 

**Additional context**

The most streams do not need to be in HD or 4k.
I don't know if I missed this feature. It is not displayed at the toolbar of the video. In this case making this option more easily accessible would be nice. :)

**Comment by CommanderStorm** [comment]
Related past issues https://github.com/TUM-Dev/gocast/issues/150 https://github.com/TUM-Dev/gocast/issues/726#issuecomment-1282233513 https://github.com/TUM-Dev/gocast/issues/727#issuecomment-1282165199

Idk if the compute situation changed, given that I heared, that some GPUs were in the process of being ordered.

**Comment by MariusAlbrecht** [comment]
you did not miss the functionality
